### ISSUE TITLE
White blood cells elevated in pericardial fluid

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10957]
**Preferred term label:**
White blood cells elevated in pericardial fluid

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Purulent pericarditis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
to be consistent with other current terms https://hpo.jax.org/browse/term/HP:0001698, 
what about this


Purulent pericardial effusion
def: Gross pus in the pericardial space or a microscopic purulent pericardial effusion.
comment: Bacterial pericarditis occurs by direct infection during trauma, thoracic surgery, or catheter drainage, by spread from an intrathoracic, myocardial, or subdiaphragmatic focus, and by hematogenous dissemination. The frequent causes are Staphylococcus and Streptococcus (rheumatic pancarditis), Haemophilus, and M. tuberculosis.
PMID: 15725041

**Comment by MickeySegal** [comment]
I wouldn't restrict the definition to "gross pus".  Although we include purulent pericarditis in synonyms for our term, we also would call this term based on elevated WBC alone.

**Comment by pnrobinson** [comment]
@MickeySegal 
modified the defintion:

Adding new term: 
- ID: HP:6001326
- label: Purulent pericardial effusion
- parents: [Pericardial effusion (HP:0001698)](https://hpo.jax.org/app/browse/term/HP:0001698)
- pmid: PMID:15725041
- def: Elevated leukocyte count in the pericardial fluid ranging from a a microscopic purulent pericardial effusion to gross pus in the pericardial space.
- comment: Bacterial pericarditis occurs by direct infection during trauma, thoracic surgery, or catheter drainage, by spread from an intrathoracic, myocardial, or subdiaphragmatic focus, and by hematogenous dissemination. The frequent causes are Staphylococcus and Streptococcus (rheumatic pancarditis), Haemophilus, and M. tuberculosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10957
- synonyms:
